### ISSUE TITLE
Message id not a SHA1 hash

### ISSUE THREAD

**Issue by bobbymiller** [pettazz/pygooglevoice#4]
I can login easily.

When I try to fetch my voicemail messages using pretty much exactly the example in example/voicemail.py I get 

  File "test_voice.py", line 11, in <module>
    print voice.voicemail().messages
  File "/mnt/refractthelight/pygooglevoice/googlevoice/util.py", line 242, in messages
    return [Message(self, *i) for i in self['messages'].items()]
  File "/mnt/refractthelight/pygooglevoice/googlevoice/util.py", line 181, in **init**
    assert is_sha1(id), 'Message id not a SHA1 hash'

AssertionError: Message id not a SHA1 hash

**Comment by jaraco** [comment]
It's not enough to comment out that line. I removed that line in 8058606, but still:

```
googlevoice master $ .tox/python/bin/python -m pdb examples/download-mp3.py
> /Users/jaraco/Dropbox/code/main/googlevoice/examples/download-mp3.py(1)<module>()
-> from googlevoice import Voice
(Pdb) c
Uncaught exception. Entering post mortem debugging
Running 'cont' or 'step' will restart the program
> /Users/jaraco/Dropbox/code/main/googlevoice/googlevoice/voice.py(250)download()
-> assert util.is_sha1(msg), 'Message id not a SHA1 hash'
(Pdb) Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/pdb.py", line 1697, in main
    pdb._runscript(mainpyfile)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/pdb.py", line 1566, in _runscript
    self.run(statement)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/bdb.py", line 585, in run
    exec(cmd, globals, locals)
  File "<string>", line 1, in <module>
  File "/Users/jaraco/Dropbox/code/main/googlevoice/examples/download-mp3.py", line 1, in <module>
    from googlevoice import Voice
  File "/Users/jaraco/Dropbox/code/main/googlevoice/examples/download-mp3.py", line 11, in run
    message.download(download_dir)
  File "/Users/jaraco/Dropbox/code/main/googlevoice/googlevoice/util.py", line 192, in download
    return self.folder.voice.download(self, adir)
  File "/Users/jaraco/Dropbox/code/main/googlevoice/googlevoice/voice.py", line 250, in download
    assert util.is_sha1(msg), 'Message id not a SHA1 hash'
AssertionError: Message id not a SHA1 hash
msg
'NZIIOVOHHMRMXOZIGWHTUMGPITQLXSUQSKGYIPVO'
```

**Comment by jaraco** [comment]
Issue is fixed in [googlevoice 2.0.3](https://pypi.org/project/googlevoice/2.0.3/).
